src/fs/bare/tree.ts

```diff
@@ -17,8 +17,9 @@ class BareTree extends BaseTree {
     this.links = links
   }
 
+  // why is this async? consistent interface? if so, maybe be explicit what we're implementing
   static async empty(): Promise<BareTree> {
-    return new BareTree({})
+    return new BareTree({}) // Why not just part of the constructor as a defautl value?
```

Comment:
Just for the consistent static interface between trees

Reply:
Looking at the other instances, we may be able to fold all of these into a constructor with named (object) arguments. Need to explore a bit more first, but it's in the running.